### PR TITLE
Image support

### LINKED ISSUE #32: As a user, I want to be able to insert images
Inserting images is an integral part of flashcard apps. It can help strengthen the memory of language learners by directly associating foreign words with concepts rather than using English as an intermediary.

Currently, images are not supported. I want to be able to add images.

Some things to do:
- [x] Display images on the flashcard view.
- [x] Modify add and edit commands to allow insertion of references to images already in the filesystem.
- [x] Remove images by specifying an empty image tag in the edit command.
- [x] Create a command to import images into _Acquizition_.
- [x] Add tests for the above methods.

### PR DESCRIPTION
**Pull request by robincyu**
Closes #32.

This PR creates fully-functional image support.

- Using the `image` command, it is possible to import images into _Acquizition_'s internal image filesystem.
- Using the `i/` prefix, it is possible to add and edit flashcards to include images inside the internal image filesystem, which are then properly displayed on the front faces of the flashcards.
- It is possible to remove an image from a flashcard by editing the flashcard and specifying an empty image prefix.

### CODE REVIEW

File: src/main/java/seedu/address/commons/core/Messages.java [hunk 1]
```diff
@@ -11,5 +11,7 @@
     public static final String MESSAGE_INVALID_PATH = "Invalid path! \n%1$s";
     public static final String MESSAGE_SYSTEM_ERROR = "System error";
     public static final String MESSAGE_FLASHCARDS_LISTED_OVERVIEW = "%1$d flashcards listed!";
+    public static final String MESSAGE_IMAGE_NOT_FOUND = "This image is not found in the database."
+        + "Did you forget to import the image?";
```
Comment (sergiovieri) [inline src/main/java/seedu/address/commons/core/Messages.java]: Should add space before this

File: src/test/java/seedu/address/logic/parser/EditCommandParserTest.java [hunk 2]
```diff
@@ -59,7 +59,7 @@ public void parse_invalidPreamble_failure() {
         assertParseFailure(parser, "1 some random string", MESSAGE_INVALID_FORMAT);
 
         // invalid prefix being parsed as preamble
-        assertParseFailure(parser, "1 i/ string", MESSAGE_INVALID_FORMAT);
+        assertParseFailure(parser, "1 h/ string", MESSAGE_INVALID_FORMAT);
```
Comment (sergiovieri) [inline src/test/java/seedu/address/logic/parser/EditCommandParserTest.java:62]: lol

File: src/main/resources/view/FlashcardCardView.fxml [hunk 3]
```diff
@@ -3,39 +3,32 @@
...
                 <font>
                     <Font size="20.0"/>
                 </font>
-                <HBox.margin>
-                    <Insets top="100.0"/>
-                </HBox.margin>
             </Label>
-        </HBox>
-        <Rectangle arcHeight="5.0" arcWidth="5.0" height="1000.0" opacity="0.02" rotate="50.0" stroke="BLACK"
-                   strokeType="INSIDE" translateX="300.0" translateY="-600.0" width="1000.0"/>
```
Comment (sergiovieri) [inline src/main/resources/view/FlashcardCardView.fxml]: erased?

**Review by sergiovieri — CHANGES_REQUESTED** [review]

File: src/test/java/seedu/address/logic/parser/EditCommandParserTest.java (hunk 2 above)
Comment (robincyu) [reply src/test/java/seedu/address/logic/parser/EditCommandParserTest.java:62]: This is actually a valid prefix now...

File: src/main/java/seedu/address/commons/core/Messages.java (hunk 1 above)
Comment (robincyu) [reply src/main/java/seedu/address/commons/core/Messages.java]: Okay, will do. Nice catch!

File: src/main/resources/view/FlashcardCardView.fxml (hunk 3 above)
Comment (robincyu) [reply src/main/resources/view/FlashcardCardView.fxml]: Yeah, the whole thing is already wrapped in a VBox. Keeping this seems to break things.

Comment (sergiovieri) [reply src/main/resources/view/FlashcardCardView.fxml]: to revert, you also need to put back the minWidth and maxWidth of -Infinity

**Review by sergiovieri — APPROVED** [review]

**Pull request merged**